ocs_ci/helpers/e2e_helpers.py

```
@@ -113,6 +153,1046 @@ def get_all_combinations_map(providers, bucket_types):
+            obj_key,
+            len(versions),
+        )
+    else:
+        remaining_version_ids = [
+            version["VersionId"]
+            for version in get_obj_versions(mcg_obj, awscli_pod, bucket_name, obj_key)
+        ]
+        raise AssertionError(
+            f"Non-current versions were not expired for {bucket_name}/{obj_key}; "
+            f"remaining versions: {remaining_version_ids}"
+        )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)